src/mito2/src/memtable/merge_tree/data.rs

```diff
@@ -662,17 +663,23 @@ impl<'a> DataPartEncoder<'a> {
         }
     }
 
-    fn writer_props(&self) -> Option<WriterProperties> {
-        self.row_group_size.map(|size| {
-            WriterProperties::builder()
-                .set_max_row_group_size(size)
-                .build()
-        })
+    // todo(hl): more customized config according to region options.
+    fn writer_props(&self) -> WriterProperties {
+        let mut builder = WriterProperties::builder();
+        if let Some(row_group_size) = self.row_group_size {
+            builder = builder.set_max_row_group_size(row_group_size)
+        }
+        builder = builder
+            .set_compression(Compression::ZSTD(ZstdLevel::default()))
+            .set_statistics_enabled(EnabledStatistics::None);
```

Comment:
Should we change the encoding of timestamp, sequence columns instead of enabling compression?

Reply:
> Should we change the encoding of timestamp, sequence columns instead of enabling compression?

Did a simple test of writing 1000w rows into DataBuffer, and the frozen data part sizes are: 
- raw: 315MB
- set column encodings for ts/pk_index/sequence/op_type: 166MB
- enable zstd: 49MB
- enable zstd & set column encodings: 25MB